package.json

```diff
@@ -15,7 +15,7 @@
   "scripts": {
     "lint": "yarn workspaces foreach -A run lint",
     "clean": "yarn workspaces foreach -A run clean",
-    "build": "yarn run clean && yarn workspaces foreach -A run build:all",
+    "build": "yarn run clean && yarn workspaces foreach -A -t run build:all",
```

Comment:
did you encounter an issue building the packages?

Reply:
I took this from Nicolas's branch, it will arrange the order of installation of the packages